test/asserts/bank-identifier-code-assert_test.js

```diff
@@ -56,11 +56,31 @@ describe('BankIdentifierCodeAssert', () => {
     }
   });
 
+  it('should throw an error if the input value is not a valid bic', () => {
+    try {
+      new Assert().BankIdentifierCode().validate('BICOLETO');
+
+      should.fail();
+    } catch (e) {
+      e.should.be.instanceOf(Violation);
+      e.value.should.equal('BICOLETO');
+    }
+  });
+
   it('should accept a valid bic without branch code', () => {
     new Assert().BankIdentifierCode().validate('FOOBARBI');
   });
 
   it('should accept a valid bic with branch code', () => {
     new Assert().BankIdentifierCode().validate('FOOBARBIXXX');
   });
+
+  it('should not be case sensitive', () => {
```

Comment:
Maybe phrase this with positive wording?

```suggestion
  it('should be case-insensitive', () => {
```